addon/record-data.js

```diff
@@ -732,13 +746,12 @@ export default class FragmentRecordData extends RecordData {
       );
     }
 
-    const changedAttributeKeys = super.pushData(data, calculateChange);
     const changedKeys = mergeArrays(changedAttributeKeys, changedFragmentKeys);
     if (gte('ember-data', '4.5.0') && changedKeys?.length > 0) {
       internalModelFor(this).notifyAttributes(changedKeys);
     }
     // on ember-data 2.8 - 4.4, InternalModel.setupData will notify
-    return changedKeys;
+    return changedKeys || [];
```

Comment:
This line fix this issue https://github.com/adopted-ember-addons/ember-data-model-fragments/issues/470

Reply:
Should this be a `??`?

Reply:
This methods should always return an array as it is actually doing a `changedKeys.length` afterward. So I prefer to be safe